### ISSUE TITLE
Some binding don't work after app resume on android

### ISSUE THREAD

**Issue by elena-p** [nativescript-vue/nativescript-vue#393]
### Version
2.0.2

### Reproduction link
[https://play.nativescript.org/?template=play-vue&id=HdDm9M&v=400](https://play.nativescript.org/?template=play-vue&id=HdDm9M&v=400)
### Platform and OS info
Android 9 / NativeScript 5

### Steps to reproduce
Open the demo playground application (https://play.nativescript.org/?template=play-vue&id=HdDm9M&v=400) in the preview app on Android

Suspend the app

Resume the app

Click the "Sign up" link at the bottom of the screen

### What is expected?
Third text field for confirm password should be shown; 
The big button should display "Sign up"; 
"Back to Login" link at the bottom of the screen should be shown; 

### What is actually happening?
Nothing - the page remains the same even though the isLoggingIn value is changing

---
I was adding a vue demo to a nativescript plugin and noticed that a text binding doesn't work after resume on android.

**Comment by yesyo** [comment]
I'm experiencing this as well. Any workaround?

**Comment by elena-p** [comment]
Hi @yesyo,

Are there any additional steps that you do in order to reproduce this issue? It used to happen every time when I logged it, however, today I am not able to reproduce it even once.

**Comment by yesyo** [comment]
Not sure. It happens on our app which runs on an Android 6 tablet. I just force quit the app in the `onPause` event as a workaround for now.

**Comment by rigor789** [comment]
@yesyo can you confirm you are running the latest version of NativeScript-Vue? (2.0.2)

This used to be an issue in prior versions, but haven't seen it happen since 2.0

**Comment by yesyo** [comment]
Yes I'm running 2.0.2. I'll try running the demo playground app on our device to see if it works. 
Thanks.

**Comment by danielrearden** [comment]
I can confirm I'm seeing the same issue on Android 8 with `nativescript-vue` at `2.0.2`. After seeing similar issues with our app, I updated the cli, started a new project and just pasted in @elena-p 's demo code. The app behaves as expected until it leaves the foreground, then it loses its reactivity. Would love to know if @yesyo or anyone else experiencing the same issue found a suitable workaround.

**Comment by Alexander-Bliznyuk** [comment]
I encountered a similar issue on `cyanogenmod` with `nativescript-vue 2.2.2`. I figured out that `topmost()` frame becomes `undefined` as soon as app suspended. So on app resume I use this as a robust indicator of the issue. Then I navigate to the current page again and this provides almost seamless experience.

```
app.android.on(app.AndroidApplication.activityResumedEvent, () => {
    if (topmost() === undefined) {
        // I save currentPage in the course of manual routing
        mainContentFrame.nativeView.navigate(currentPage);
    }
 });
```
